lib/util.js

```diff
@@ -149,3 +149,22 @@ module.exports.log = internals.consoleFn('log');
 
 
 module.exports.dir = internals.consoleFn('dir');
+
+module.exports.hash = function (values = {}) {
```

Comment:
Since this is only called in two places, and never with args, we should just do `Object.create(null)` inline.